src/os/bluestore/BlueStore.cc

```diff
@@ -2133,7 +2133,7 @@ void BlueStore::ExtentMap::reshard(
 	     << needs_reshard_end << ")" << std::dec << dendl;
   }
 
-  fault_range(db, needs_reshard_begin, needs_reshard_end);
+  fault_range(db, needs_reshard_begin, (needs_reshard_end-needs_reshard_begin));
```

Comment:
Good catch!  Can you add a space before and after the "-" operator?
Also there are some rules for making a valid Pull Request, please refer to https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst for details.